src/ui/ProjectsFolderBrowser.cpp

```diff
@@ -574,3 +595,9 @@ ProjectsFolderBrowser::_CompareProjectItems(const BListItem* a, const BListItem*
 		
 	return 0;
 }
+
+void
+ProjectsFolderBrowser::SelectionChanged() {
```

Comment:
This could be managed by the selection message received by the window.. but it's OK.. eventually it will change when my ActionManager will be finish

Reply:
Yes I've broken the encapsulation here... I assumed that the target for the selection message was the ProjectBrowser

Reply:
leaving this as is, then.